Add files via upload

johnmckellar opened the pull request:
1.	How do we deal with hyphens in cases such as with the Clarence-Moreton Basin? dct:hasPart data:CLARENCE_MORETON BASIN , ???

Also:
dct:hasPart data:PALMER_BARRON_SUBPROVINCE , (for the Palmer-Barron Subprovince)

and

dct:isPartOf data:CONNORS_AUBURN_PROVINCE , (for Connors-Auburn)

2.	When beginning a line with “””, does the line need to end with the same “””? I thought that it was necessary, but Nick did not do this in this file, but there were only one or two cases of same. Not sure if this was an oversight or not. I know that Nick had not finished working on the file.

3.	dct:alternative EASTERN_FOLD_BELT , EASTERN_SUCCESSION , <<< In cases such as this, do we need to repeat ‘dct:alternative’ for the EASTERN_SUCCESSION ,

Also here: We are not looking at an alternative term, but a non-preferred term. Is there other dct code that would be better suited? I couldn’t find one.

4.	Ditto point 3 for:
dct:hasPart data:EROMANGA_BASIN , SURAT_BASIN , etc.

5.	Is it possible to give the following, for example, as a time range, in this case Llandovery to Frasnain? Here, we would have maximum and minimum ages given.

     time:hasTime ischart:Frasnian , ischart:MiddleDevonian , ischart:LowerDevonian , ischart:Pridoli , ischart:Ludlow ,  ischart:Wenlock , ischart:Llandovery ;

This would be instead of listing all of the above. This is more of an issue for igneous provinces where magmatic activity was episodic and where the full range of associated rocks have not been isotopically dated.

6.	For the following, note that the Great Australian Superbasin (GASB) contains the listed basins, which fall under RHAETIAN_JURASSIC_CRETACEOUS_BASIN_TYPES

data:GreatAustralianSuperbasin dct:hasPart data:EROMANGA_BASIN , SURAT_BASIN , MULGILDIE_BASIN , CARPENTARIA_BASIN , CLARENCE_MORETON BASIN , NAMBOUR_BASIN , LAURA_BASIN , dct:isPartOf data:RHAETIAN_JURASSIC_CRETACEOUS_BASIN_TYPES , 

And there are basins listed under RHAETIAN_JURASSIC_CRETACEOUS_BASIN_TYPES , 
But not all basins listed under the latter are embraced by the GASB (such as the Stanwell, Styx, Baga). Is this an issue?

Note that I have introduced a class of ‘Basin Types’ (I could not find a better term), as not all basins are simply basins. So, under Basin Types, there are basins, troughs, depressions, grabens, half-grabens, etc. The Features Type vocab would need to be amended, if this proposal is acceptable.

7.	data:AdavaleBasin
        a geof:Basin ,
        geo:Feature ,
        sdo:Landform ;
I note that Nick did not have a comma at the end of the first line above, but, as I have added to such lines, I have placed a comma at the end of the line.

8.	time:hasTime ischart:Silurian,
        ischart:Ordovician,
        ischart:Cambrian,
        ischart:Neoproterozoic ;
For the above, I notice that Nick did not put spaces between the time-rock unit (e.g., Ordovician) and the following comma in the main body of the file,

but, down the bottom of the file, where he had basins with non-spatial info listed, he consistently had, for each, a space between the time-rock unit (e.g., Paleogene) and the comma: e.g.:
data:WoorabindaBasin 
    a geof:Basin , geo:Feature , sdo:Landform ;
    time:hasTime ischart:Paleogene , ischart:Neogene ;    
    sdo:name "Woorabinda Basin"@en ;

Why the difference. Does it matter or not? For the moment, I have left this ‘as is’.

9.	Do we need to introduce MERLIN KEY CODE numbers or the equivalent into this file? And also link where we can to GA’s Provinces? Nick is working for GA on this anyway and will be looking at all linkages from their end.

10.	data:BagaBasin dct:isPartOf data:data:RHAETIAN_JURASSIC_CRETACEOUS_BASIN_TYPES , dct:alternative JIM_CROW_BASIN , dct:description """The basin contains the Baga beds (Geoscience-Australia Stratigraphic Number 81555—Reserved Status, Australian Stratigraphic Units Database), renamed from the Jim Crow beds. The Jim Crow Basin, associated with the latter unit, is proposed to be renamed the Baga Basin (Paul Blake, pers. comm., 29.09.2020). ,

Note that I have a full stop and a comma at the end of the line??!!

johnmckellar committed:
Add files via upload

1.	How do we deal with hyphens in cases such as with the Clarence-Moreton Basin? dct:hasPart data:CLARENCE_MORETON BASIN , ???

Also:
dct:hasPart data:PALMER_BARRON_SUBPROVINCE , (for the Palmer-Barron Subprovince)

and

dct:isPartOf data:CONNORS_AUBURN_PROVINCE , (for Connors-Auburn)

2.	When beginning a line with “””, does the line need to end with the same “””? I thought that it was necessary, but Nick did not do this in this file, but there were only one or two cases of same. Not sure if this was an oversight or not. I know that Nick had not finished working on the file.

3.	dct:alternative EASTERN_FOLD_BELT , EASTERN_SUCCESSION , <<< In cases such as this, do we need to repeat ‘dct:alternative’ for the EASTERN_SUCCESSION ,

Also here: We are not looking at an alternative term, but a non-preferred term. Is there other dct code that would be better suited? I couldn’t find one.

4.	Ditto point 3 for:
dct:hasPart data:EROMANGA_BASIN , SURAT_BASIN , etc.

5.	Is it possible to give the following, for example, as a time range, in this case Llandovery to Frasnain? Here, we would have maximum and minimum ages given.

     time:hasTime ischart:Frasnian , ischart:MiddleDevonian , ischart:LowerDevonian , ischart:Pridoli , ischart:Ludlow ,  ischart:Wenlock , ischart:Llandovery ;

This would be instead of listing all of the above. This is more of an issue for igneous provinces where magmatic activity was episodic and where the full range of associated rocks have not been isotopically dated.

6.	For the following, note that the Great Australian Superbasin (GASB) contains the listed basins, which fall under RHAETIAN_JURASSIC_CRETACEOUS_BASIN_TYPES

data:GreatAustralianSuperbasin dct:hasPart data:EROMANGA_BASIN , SURAT_BASIN , MULGILDIE_BASIN , CARPENTARIA_BASIN , CLARENCE_MORETON BASIN , NAMBOUR_BASIN , LAURA_BASIN , dct:isPartOf data:RHAETIAN_JURASSIC_CRETACEOUS_BASIN_TYPES , 

And there are basins listed under RHAETIAN_JURASSIC_CRETACEOUS_BASIN_TYPES , 
But not all basins listed under the latter are embraced by the GASB (such as the Stanwell, Styx, Baga). Is this an issue?

Note that I have introduced a class of ‘Basin Types’ (I could not find a better term), as not all basins are simply basins. So, under Basin Types, there are basins, troughs, depressions, grabens, half-grabens, etc. The Features Type vocab would need to be amended, if this proposal is acceptable.

7.	data:AdavaleBasin
        a geof:Basin ,
        geo:Feature ,
        sdo:Landform ;
I note that Nick did not have a comma at the end of the first line above, but, as I have added to such lines, I have placed a comma at the end of the line.

8.	time:hasTime ischart:Silurian,
        ischart:Ordovician,
        ischart:Cambrian,
        ischart:Neoproterozoic ;
For the above, I notice that Nick did not put spaces between the time-rock unit (e.g., Ordovician) and the following comma in the main body of the file,

but, down the bottom of the file, where he had basins with non-spatial info listed, he consistently had, for each, a space between the time-rock unit (e.g., Paleogene) and the comma: e.g.:
data:WoorabindaBasin 
    a geof:Basin , geo:Feature , sdo:Landform ;
    time:hasTime ischart:Paleogene , ischart:Neogene ;    
    sdo:name "Woorabinda Basin"@en ;

Why the difference. Does it matter or not? For the moment, I have left this ‘as is’.

9.	Do we need to introduce MERLIN KEY CODE numbers or the equivalent into this file? And also link where we can to GA’s Provinces? Nick is working for GA on this anyway and will be looking at all linkages from their end.

10.	data:BagaBasin dct:isPartOf data:data:RHAETIAN_JURASSIC_CRETACEOUS_BASIN_TYPES , dct:alternative JIM_CROW_BASIN , dct:description """The basin contains the Baga beds (Geoscience-Australia Stratigraphic Number 81555—Reserved Status, Australian Stratigraphic Units Database), renamed from the Jim Crow beds. The Jim Crow Basin, associated with the latter unit, is proposed to be renamed the Baga Basin (Paul Blake, pers. comm., 29.09.2020). ,

Note that I have a full stop and a comma at the end of the line??!!

katewd commented on dataset.ttl:
```diff
@@ -535,10 +575,11 @@ data:GalileeBasin a geof:Basin ,
         ischart:Pennsylvanian,
         ischart:UpperMississippian ;
-        #skos:scopeNote "That the Galilee Basin section in the Koburra Trough extends down into the Upper Mississippian is uncertain."@en ;#
+        # How do we add a note "That the Galilee Basin section in the Koburra Trough extends down into the Upper Mississippian is uncertain."@en ;#
```
Hi John, Can you please edit line 578 of your new file to remove the text "#How do we add a note" and replace it with "#skos:scopeNote"  Thanks.

DavidCrosswellGSQ replied:
@johnmckellar - see https://www.w3.org/TR/swbp-skos-core-guide/#secdocumentation

johnmckellar replied:
Thanks Kate. I have modified the dataset for all comments and will upload it again.

katewd commented on dataset.ttl:
```diff
@@ -856,10 +912,10 @@ data:NogoSubprovince
             geox:hasRole grole:bounding-box ] ;
     time:hasTime ischart:Cisuralian,
         ischart:UpperDevonian ;
-        #Can I add a note, similar to a scopeNote, to query the Upper Devonian assignment? I need to find out why there is no SKOS terminology here.#
```
This comment will need to be removed too, when you have the terminology, thanks

KellyVance commented:
There are a number of anomalous characters throughout the file that will need a find and replace
e.g. sdo:name "Paleoproterozoic�Mesoproterozoic Basins"@en .

They all _look_ like they should be hypens

katewd commented on dataset.ttl:
```diff
@@ -1071,7 +1134,9 @@ data:TexasSubprovince
 
-data:ThomsonOrogen a geof:Orogen ,
+data:ThomsonOrogen dct:hasPart data:NeoproterozoicLowerPaleozoicProvinces , data:NeoproterozoicLowerPaleozoicBasinTypes , data:MACROSSAN_IGNEOUS_PROVINCE , data:PAMA_IGNEOUS_PROVINCE , data:KENNEDY_IGNEOUS_PROVINCE , data:IRON_RANGE_PROVINCE , data:GREENVALE_PROVINCE , data:CHARTERS_TOWERS_PROVINCE , data:BARNARD_PROVINCE , data:WARBURTON_BASIN , dct:isPartOf data:TASMANIDES
+# TO CHECK THIS OUT WITH DAVE & BOB. check out all Tasmanides components #
```
These comments are fine while you are working on the file John, but please go through and remove them once you have the answers you need, so that the 'final-final' version is clean

katewd commented on dataset.ttl:
```diff
@@ -1126,21 +1193,22 @@ data:WesternSubprovince
         sdo:Landform ,
-        geof:Province ;
+        geof:IgneousProvince ;
+         dct:description """In the Geology of Queensland (Jell 2013), this feature has been referred to both as the Whitsunday Volcanic Province and simply as the Whitsunday Province. But, as it contains a significant complement of intrusive rocks, it is here and better referred to as the Whitsunday Igneous Province (David Purdy, personal communication).
```
remove this comment when completed

katewd commented on dataset.ttl:
```diff
@@ -1172,7 +1240,7 @@ data:YarramanSubprovince
         dcat:Dataset ;
     dct:description """This is a Linked Data dataset (i.e. online in HTML & RDF, according to LD principles) of the Geological Features of Interest of Queensland.
```
is this description correct?

katewd commented on dataset.ttl:
```diff
@@ -1212,9 +1280,9 @@ This tool uses the SWEET Geological Features Profile Ontology (https://github.co
     dcat:servesDataset <http://linked.data.gov.au/dataset/qldgeofeatures> .
 
-
+        # in here twice >>>
```
to be removed once file is finished

katewd commented on dataset.ttl:
```diff
@@ -1224,188 +1292,437 @@ data:YarramanSubprovince
-        ischart:UpperDevonian ;
+        ischart:Devonian ;
     sdo:name "Yarraman Subprovince"@en .    
+   <<< in here twice; comapre both entries #
```
Delete the duplicate entry please John, whichever is the lesser or has the least info, or the second copy if they are exactly the same

katewd commented on dataset.ttl:
```diff
@@ -1224,188 +1292,437 @@ data:YarramanSubprovince
+.
+
+data:MountIsaProvince dct:hasPart data:YARINGA_SUBPROVINCE , data:KALKADOON-LEICHARDT_SUBPROVINCE , data:WESTERN_SUBPROVINCE , data:EASTERN_SUBPROVINCE , dct:isPartOf data:NORTH_AUSTRALIAN_CRATON , dct:isPartOf data:PaleoproterozoicMesoproterozoicProvinces ,
+# ??KALKADOON-LEICHARDT with hyphen or otherwise?? #
```
comment to be removed once resolved

katewd commented on dataset.ttl:
```diff
@@ -1224,188 +1292,437 @@ data:YarramanSubprovince
+.
+
+data:KennedyIgneousProvince dct:hasPart data:JARDINE_SUBPROVINCE , data:DAINTREE_SUBPROVINCE , data:HERBERTON_SUBPROVINCE , data:LAKEFIELD_SUBPROVINCE , data:TATE_SUBPROVINCE , data:KIDSTON_SUBPROVINCE , data:KANGAROO_HILLS_SUBPROVINCE , data:PALUMA_SUBPROVINCE , data:BURDEKIN_FALLS_SUBPROVINCE , data:TORRES_STRAIT_SUBPROVINCE , KIDSTON_SUBPROVINCE , dct:alternative KENNEDY_PROVINCE , KENNEDY_IGN ...
+# tO CHECK IF THIS IS CODED/MODELLED OK: KIP is part of Mossman, Thomson & NAC, and the inverse is true, but the Mossman, Thomson and NAC don't bear the same relationships #
```
comment to be removed once resolved

KellyVance commented on dataset.ttl:
```diff
@@ -32,10 +32,10 @@ data:AdavaleBasin
 
-data:AnakieProvince
+data:AnakieProvince dct:hasPart data:FORK_LAGOONS_SUBPROVINCE , dct:isPartOf data:THOMSON_OROGEN , data:NEOPROTEROZOIC_LOWER_PALEOZOIC_PROVINCES ,
         a geo:Feature ,
         sdo:Landform ,
-        geof:Province ;
+        geof:TectonisedMetomorphosedProvince ;
```
The syntax here is giving me validation errors and I think this format is repeated throughout the file on multiple objects.

I believe it needs to look more like the below, declaring what the object is e.g. feature and landform then providing the dct:hasPart and isPartOf as semi-colon punctuated declarations

data:AnakieProvince 
        a geo:Feature ,
        sdo:Landform ,
        geof:TectonisedMetomorphosedProvince ;
    dct:hasPart data:FORK_LAGOONS_SUBPROVINCE ;
    dct:isPartOf data:THOMSON_OROGEN , data:NEOPROTEROZOIC_LOWER_PALEOZOIC_PROVINCES ;
    geo:hasGeometry [ a geo:Geometry ;

johnmckellar replied:
a typo: TectonisedMetomorphosedProvince
It's in there so many times, and I cannot believe that I have overlooked it before now!!

Semicolons inserted.

katewd commented on dataset.ttl:
```diff
@@ -1224,188 +1292,437 @@ data:YarramanSubprovince
+        geo:Feature ,
+        sdo:Landform ;
+    time:hasTime ischart:Quaternary , ischart:Neogene , ischart:Paleogene , ischart:UpperCretaceous ;
+    sdo:name "Upper Cretaceous � Cenozoic (largely Paleogene�Neogene) Basins"@en .
```
are these symbols supposed to be hyphens here?

johnmckellar replied:
Kate,
In the file, they are indeed hyphens. So, why are they showing up as �?

katewd reviewed (changes requested):
Hi John, 
Can you please edit line 578 of your new file to remove the text "#How do we add a note" and replace it with "#skos:scopeNote" 
Once you've done that, can you please re-upload the file to your branch here and I will merge it into the master branch. I note you've got some comments to yourself as reminders within the file, and this is fine while you are still working on it. Once you have finished completely everything you want to change, we'll just need to go through and make sure all of the comments have been removed. 
Great work, you added a lot of data into this file and it looks good to me.
Thanks.

KellyVance reviewed (changes requested):
The syntax of each feature will need to be edited as per the comment on the Anakie Province to ensure the file gets read correctly.

the special characters will need to be replaced.

the working notes need to be deleted from the file before it can go into the master branch

KellyVance commented:
Hi @johnmckellar, the below is a 'to the best of my knowledge' answer. I'm still getting a complete handle on some of the logic and syntax myself. So my answer may require a little input or correction from @katewd.

Apologies for the delay in responding directly to this list.


> 1. How do we deal with hyphens in cases such as with the Clarence-Moreton Basin? dct:hasPart data:CLARENCE_MORETON BASIN , ???
> 
> Also:
> dct:hasPart data:PALMER_BARRON_SUBPROVINCE , (for the Palmer-Barron Subprovince)
> 
> and
> 
> dct:isPartOf data:CONNORS_AUBURN_PROVINCE , (for Connors-Auburn)
> 
_This shouldn't matter too much in the URIs as long as we are consistent. PALMER_BARRON_SUBPROVINCE or PALMER-BARRON-SUBPROVINCE or PalmerBarronSubProvince are all fine for a URI as long as the prefLabel or title has the 'correct' name._

> 2. When beginning a line with “””, does the line need to end with the same “””? I thought that it was necessary, but Nick did not do this in this file, but there were only one or two cases of same. Not sure if this was an oversight or not. I know that Nick had not finished working on the file.

_As far as I am aware, Yes. If you declare the start of a text block with """ you should end it the same way_

> 3. dct:alternative EASTERN_FOLD_BELT , EASTERN_SUCCESSION , <<< In cases such as this, do we need to repeat ‘dct:alternative’ for the EASTERN_SUCCESSION ,

_If it falls under the same declaration (e.g. dct:alternative) all instances of that type can simply be separated by a comma without repeating the type. Just make sure you end the group that sit in that type with a semi-colon_
> 
> Also here: We are not looking at an alternative term, but a non-preferred term. Is there other dct code that would be better suited? I couldn’t find one.

_Not sure, sorry._
> 
> 4. Ditto point 3 for:
>    dct:hasPart data:EROMANGA_BASIN , SURAT_BASIN , etc.

_Yes. 

It would look like
dct:hasPart data:this, data:that ;
dct:isPartOf data:theOther ;_

> 5. Is it possible to give the following, for example, as a time range, in this case Llandovery to Frasnain? Here, we would have maximum and minimum ages given.
>    time:hasTime ischart:Frasnian , ischart:MiddleDevonian , ischart:LowerDevonian , ischart:Pridoli , ischart:Ludlow ,  ischart:Wenlock , ischart:Llandovery ;
> 
> This would be instead of listing all of the above. This is more of an issue for igneous provinces where magmatic activity was episodic and where the full range of associated rocks have not been isotopically dated.
> 
_Not sure. you'd have to ask Nick on the specifics of how date ranges are handled. I'm not sure the background logic and inferences that the system would make._

> 6. For the following, note that the Great Australian Superbasin (GASB) contains the listed basins, which fall under RHAETIAN_JURASSIC_CRETACEOUS_BASIN_TYPES
> 
> data:GreatAustralianSuperbasin dct:hasPart data:EROMANGA_BASIN , SURAT_BASIN , MULGILDIE_BASIN , CARPENTARIA_BASIN , CLARENCE_MORETON BASIN , NAMBOUR_BASIN , LAURA_BASIN , dct:isPartOf data:RHAETIAN_JURASSIC_CRETACEOUS_BASIN_TYPES ,
> 
> And there are basins listed under RHAETIAN_JURASSIC_CRETACEOUS_BASIN_TYPES ,
> But not all basins listed under the latter are embraced by the GASB (such as the Stanwell, Styx, Baga). Is this an issue?

_If not all the GASB basins fit within the RJCBTs (i.e. it is an overlaps rather than a contains) I would remove the dct:isPartOf from data:GreatAustralianSuperbasin and declare all the basins individually, so you don't imply all RJCBTs fit in the GASB_
> 
> Note that I have introduced a class of ‘Basin Types’ (I could not find a better term), as not all basins are simply basins. So, under Basin Types, there are basins, troughs, depressions, grabens, half-grabens, etc. The Features Type vocab would need to be amended, if this proposal is acceptable.
> 
> 7. data:AdavaleBasin
>    a geof:Basin ,
>    geo:Feature ,
>    sdo:Landform ;
>    I note that Nick did not have a comma at the end of the first line above, but, as I have added to such lines, I have placed a comma at the end of the line.

_The above is correct. Helps to think in natural language here and see the correlation.
the AdavaleBasin is a Basin, Feature, and Landform.
(the) data:AdavaleBasin (is) a geof:Basin ,geo:Feature , sdo:Landform ;_


>8. time:hasTime ischart:Silurian,
>    ischart:Ordovician,
>    ischart:Cambrian,
>    ischart:Neoproterozoic ;
>    For the above, I notice that Nick did not put spaces between the time-rock unit (e.g., Ordovician) and the following comma in the main body of the file,
> 
> but, down the bottom of the file, where he had basins with non-spatial info listed, he consistently had, for each, a space between the time-rock unit (e.g., Paleogene) and the comma: e.g.:
> data:WoorabindaBasin
> a geof:Basin , geo:Feature , sdo:Landform ;
> time:hasTime ischart:Paleogene , ischart:Neogene ;
> sdo:name "Woorabinda Basin"@en ;
> 
> Why the difference. Does it matter or not? For the moment, I have left this ‘as is’.

_Doesn't really matter. It is not super important but it is good practice to be consistent throughout._
> 
>9. Do we need to introduce MERLIN KEY CODE numbers or the equivalent into this file? And also link where we can to GA’s Provinces? Nick is working for GA on this anyway and will be looking at all linkages from their end.

_Again, I dont have the answer on this. Sorry. I don't see why we would migrate obsolete MERLIN codes, but Nick may have had a reason. Best to check with him before you omit or delete them._

> 10. data:BagaBasin dct:isPartOf data:data:RHAETIAN_JURASSIC_CRETACEOUS_BASIN_TYPES , dct:alternative JIM_CROW_BASIN , dct:description """The basin contains the Baga beds (Geoscience-Australia Stratigraphic Number 81555—Reserved Status, Australian Stratigraphic Units Database), renamed from the Jim Crow beds. The Jim Crow Basin, associated with the latter unit, is proposed to be renamed the Baga Basin (Paul Blake, pers. comm., 29.09.2020). ,
> 
> Note that I have a full stop and a comma at the end of the line??!!

_I think this should be:
```
data:BagaBasin a geof:Basin, 
      geo:Feature, 
      sdo:Landform ;   
   dct:isPartOf data:RHAETIAN_JURASSIC_CRETACEOUS_BASIN_TYPES ;
   dct:alternative JIM_CROW_BASIN ;
   dct:description """The basin contains the Baga beds (Geoscience-Australia Stratigraphic Number 81555, Reserved Status, Australian Stratigraphic Units Database), renamed from the Jim Crow beds. The Jim Crow Basin, associated with the latter unit, is proposed to be renamed the Baga Basin (Paul Blake, pers. comm., 29.09.2020).""" ;
   time:hasTime ischart:LowerCretaceous ;
   sdo:name "Baga Basin"@en  .
```_

johnmckellar commented:
Vance, thanks for the comments and feedback. 👍
________________________________
From: KellyVance <notifications@github.com>
Sent: Tuesday, 20 October 2020 11:48 AM
To: geological-survey-of-queensland/qldgeofeatures-dataset <qldgeofeatures-dataset@noreply.github.com>
Cc: MCKELLAR John <John.McKellar@dnrme.qld.gov.au>; Mention <mention@noreply.github.com>
Subject: Re: [geological-survey-of-queensland/qldgeofeatures-dataset] Add files via upload (#8)


Hi @johnmckellar<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_johnmckellar&d=DwMFaQ&c=tpTxelpKGw9ZbZ5Dlo0lybSxHDHIiYjksG4icXfalgk&r=7PGzJsKGfMQg6r_oMXGYtVoxRstXoRB0iYsyk0FLe8U&m=uqQOqxXI8CRc2XBqB2IzIPsGo0JIpF4YUFFA_47OvSE&s=WLBkh1TJUx5aRb9FUiZAGgKL1jDgiYFwMdF_GavdQwM&e=>, the below is a 'to the best of my knowledge' answer. I'm still getting a complete handle on some of the logic and syntax myself. So my answer may require a little input or correction from @katewd<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_katewd&d=DwMFaQ&c=tpTxelpKGw9ZbZ5Dlo0lybSxHDHIiYjksG4icXfalgk&r=7PGzJsKGfMQg6r_oMXGYtVoxRstXoRB0iYsyk0FLe8U&m=uqQOqxXI8CRc2XBqB2IzIPsGo0JIpF4YUFFA_47OvSE&s=tN5YNj6k2iZMe7CtqsvblvABBHzxlqRzGb7i11x-M7E&e=>.

Apologies for the delay in responding directly to this list.

  1.  How do we deal with hyphens in cases such as with the Clarence-Moreton Basin? dct:hasPart data:CLARENCE_MORETON BASIN , ???

Also:
dct:hasPart data:PALMER_BARRON_SUBPROVINCE , (for the Palmer-Barron Subprovince)

and

dct:isPartOf data:CONNORS_AUBURN_PROVINCE , (for Connors-Auburn)

This shouldn't matter too much in the URIs as long as we are consistent. PALMER_BARRON_SUBPROVINCE or PALMER-BARRON-SUBPROVINCE or PalmerBarronSubProvince are all fine for a URI as long as the prefLabel or title has the 'correct' name.

  1.  When beginning a line with “””, does the line need to end with the same “””? I thought that it was necessary, but Nick did not do this in this file, but there were only one or two cases of same. Not sure if this was an oversight or not. I know that Nick had not finished working on the file.

As far as I am aware, Yes. If you declare the start of a text block with """ you should end it the same way

  1.  dct:alternative EASTERN_FOLD_BELT , EASTERN_SUCCESSION , <<< In cases such as this, do we need to repeat ‘dct:alternative’ for the EASTERN_SUCCESSION ,

If it falls under the same declaration (e.g. dct:alternative) all instances of that type can simply be separated by a comma without repeating the type. Just make sure you end the group that sit in that type with a semi-colon

Also here: We are not looking at an alternative term, but a non-preferred term. Is there other dct code that would be better suited? I couldn’t find one.

Not sure, sorry.

  1.  Ditto point 3 for:
dct:hasPart data:EROMANGA_BASIN , SURAT_BASIN , etc.

_Yes.

It would look like
dct:hasPart data:this, data:that ;
dct:isPartOf data:theOther ;_

  1.  Is it possible to give the following, for example, as a time range, in this case Llandovery to Frasnain? Here, we would have maximum and minimum ages given.
time:hasTime ischart:Frasnian , ischart:MiddleDevonian , ischart:LowerDevonian , ischart:Pridoli , ischart:Ludlow , ischart:Wenlock , ischart:Llandovery ;

This would be instead of listing all of the above. This is more of an issue for igneous provinces where magmatic activity was episodic and where the full range of associated rocks have not been isotopically dated.

Not sure. you'd have to ask Nick on the specifics of how date ranges are handled. I'm not sure the background logic and inferences that the system would make.

  1.  For the following, note that the Great Australian Superbasin (GASB) contains the listed basins, which fall under RHAETIAN_JURASSIC_CRETACEOUS_BASIN_TYPES

data:GreatAustralianSuperbasin dct:hasPart data:EROMANGA_BASIN , SURAT_BASIN , MULGILDIE_BASIN , CARPENTARIA_BASIN , CLARENCE_MORETON BASIN , NAMBOUR_BASIN , LAURA_BASIN , dct:isPartOf data:RHAETIAN_JURASSIC_CRETACEOUS_BASIN_TYPES ,

And there are basins listed under RHAETIAN_JURASSIC_CRETACEOUS_BASIN_TYPES ,
But not all basins listed under the latter are embraced by the GASB (such as the Stanwell, Styx, Baga). Is this an issue?

If not all the GASB basins fit within the RJCBTs (i.e. it is an overlaps rather than a contains) I would remove the dct:isPartOf from data:GreatAustralianSuperbasin and declare all the basins individually, so you don't imply all RJCBTs fit in the GASB

Note that I have introduced a class of ‘Basin Types’ (I could not find a better term), as not all basins are simply basins. So, under Basin Types, there are basins, troughs, depressions, grabens, half-grabens, etc. The Features Type vocab would need to be amended, if this proposal is acceptable.

  1.  data:AdavaleBasin
a geof:Basin ,
geo:Feature ,
sdo:Landform ;
I note that Nick did not have a comma at the end of the first line above, but, as I have added to such lines, I have placed a comma at the end of the line.

The above is correct. Helps to think in natural language here and see the correlation.
the AdavaleBasin is a Basin, Feature, and Landform.
(the) data:AdavaleBasin (is) a geof:Basin ,geo:Feature , sdo:Landform ;

  1.  time:hasTime ischart:Silurian,
ischart:Ordovician,
ischart:Cambrian,
ischart:Neoproterozoic ;
For the above, I notice that Nick did not put spaces between the time-rock unit (e.g., Ordovician) and the following comma in the main body of the file,

but, down the bottom of the file, where he had basins with non-spatial info listed, he consistently had, for each, a space between the time-rock unit (e.g., Paleogene) and the comma: e.g.:
data:WoorabindaBasin
a geof:Basin , geo:Feature , sdo:Landform ;
time:hasTime ischart:Paleogene , ischart:Neogene ;
sdo:name "Woorabinda Basin"@en<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_en&d=DwMFaQ&c=tpTxelpKGw9ZbZ5Dlo0lybSxHDHIiYjksG4icXfalgk&r=7PGzJsKGfMQg6r_oMXGYtVoxRstXoRB0iYsyk0FLe8U&m=uqQOqxXI8CRc2XBqB2IzIPsGo0JIpF4YUFFA_47OvSE&s=EGzTgDvW0Bv-toKABhaj9IUhRB2Nt12aAb_j5DymjfE&e=> ;

Why the difference. Does it matter or not? For the moment, I have left this ‘as is’.

Doesn't really matter. It is not super important but it is good practice to be consistent throughout.

  1.  Do we need to introduce MERLIN KEY CODE numbers or the equivalent into this file? And also link where we can to GA’s Provinces? Nick is working for GA on this anyway and will be looking at all linkages from their end.

Again, I dont have the answer on this. Sorry. I don't see why we would migrate obsolete MERLIN codes, but Nick may have had a reason. Best to check with him before you omit or delete them.

  1.  data:BagaBasin dct:isPartOf data:data:RHAETIAN_JURASSIC_CRETACEOUS_BASIN_TYPES , dct:alternative JIM_CROW_BASIN , dct:description """The basin contains the Baga beds (Geoscience-Australia Stratigraphic Number 81555—Reserved Status, Australian Stratigraphic Units Database), renamed from the Jim Crow beds. The Jim Crow Basin, associated with the latter unit, is proposed to be renamed the Baga Basin (Paul Blake, pers. comm., 29.09.2020). ,

Note that I have a full stop and a comma at the end of the line??!!

_I think this should be:

data:BagaBasin a geof:Basin,
      geo:Feature,
      sdo:Landform ;
   dct:isPartOf data:RHAETIAN_JURASSIC_CRETACEOUS_BASIN_TYPES ;
   dct:alternative JIM_CROW_BASIN ;
   dct:description """The basin contains the Baga beds (Geoscience-Australia Stratigraphic Number 81555, Reserved Status, Australian Stratigraphic Units Database), renamed from the Jim Crow beds. The Jim Crow Basin, associated with the latter unit, is proposed to be renamed the Baga Basin (Paul Blake, pers. comm., 29.09.2020).""" ;
   time:hasTime ischart:LowerCretaceous ;
   sdo:name "Baga Basin"@en  .
```_




—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_geological-2Dsurvey-2Dof-2Dqueensland_qldgeofeatures-2Ddataset_pull_8-23issuecomment-2D712536403&d=DwMFaQ&c=tpTxelpKGw9ZbZ5Dlo0lybSxHDHIiYjksG4icXfalgk&r=7PGzJsKGfMQg6r_oMXGYtVoxRstXoRB0iYsyk0FLe8U&m=uqQOqxXI8CRc2XBqB2IzIPsGo0JIpF4YUFFA_47OvSE&s=c5jHj4KJb-AoAgKD8PtpgHbp4v4YKjA9lf0JEr7G2eI&e=>, or unsubscribe<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_AMXRW2X42X7UIM6SEB4XMV3SLTT5VANCNFSM4R7ZOLGQ&d=DwMFaQ&c=tpTxelpKGw9ZbZ5Dlo0lybSxHDHIiYjksG4icXfalgk&r=7PGzJsKGfMQg6r_oMXGYtVoxRstXoRB0iYsyk0FLe8U&m=uqQOqxXI8CRc2XBqB2IzIPsGo0JIpF4YUFFA_47OvSE&s=FEiQ4Uu4aTe3VoOSKV5pGOWsztfOGGKLhL-2usrZSo0&e=>.

------------------------------
The information in this email together with any attachments is intended only for the person or entity to which it is addressed and may contain confidential and/or privileged material. There is no waiver of any confidentiality/privilege by your inadvertent receipt of this material. 
Any form of review, disclosure, modification, distribution and/or publication of this email message is prohibited, unless as a necessary part of Departmental business.
If you have received this message in error, you are asked to inform the sender as quickly as possible and delete this message and any copies of this message from your computer and/or your computer system network.
------------------------------

johnmckellar committed:
Add files via upload

Hi Everyone,
•	Where I have indicated an altLabel, I have followed this with a prefLabel, but is the latter necessary when we also give a ‘sdo:name’?
•	Note: The following links, inserted by Nick, no longer work:
•	http://geofeatures.gsq.cat/data.ttl
•	http://geofeatures.gsq.cat
•	The ‘Container of Features’ (https://gsq.cat/qldgeofeatures/feature/), and the various containers thereunder (Provinces, Subprovinces, basins, etc.), were set up (by Nick?) as a demo system. Does anyone have any ideas about this and where it has headed or should be heading?
•	BasinType will need to be introduced into the features type vocab, as will a couple of other concepts, but I will await feedback on BasinTypes first. This will embrace not only basins, but also grabens, depressions, and the like. Or is there another way of grouping basins of the same or similar age under headings without resorting to this?
Many thanks,
John

johnmckellar committed:
Add files via upload

Hi Vance, Kate, David, 
Paul has reviewed the file and found it, at least the geological part of it, in good order, with the suggested addition of a Frasnian age to two features. I have followed through on this, and, thus, the file is available for your review of the syntax. There will need to be a couple of modifications to the features type vocab, to embrace some of my additions to the file, such as the basin-types groupings. If there is a better way of approaching these groupings, please advise. These basin groups will need to appear as such in GeoResGlobe, as will other groupings, but all of this may need some tweaking.

DavidCrosswellGSQ reviewed (commented):
@johnmckellar - there are some bad characters in that file. I recommend the VSCode extension Highlight Bad Chars by Kevin Wenger.
They are dodgy hyphens.

DavidCrosswellGSQ commented:
@johnmckellar - is it worth smoothing some of the geojson? Some of it is quite verbose.
Ok if you want to leave it at that granularity.

johnmckellar commented:
Parts of it need to be left as is, as they require explanation: reasons why, etc. We can look to contract it further along.

If you think that this is verbose, then you are in for a shock when you see the biostrat (most of it introduced into the notes). I am putting this info in now so that I don't, or someone else doesn't, have to locate it again. In fact. there is no one else who can do it anyway.  Some of it is new info that has not seen the light of day, especially in placing numerical/isotopic ages against zonal boundaries (presently for the early and mid-Permian).  It has taken time as the info is derived from multiple sources, some of which that no one else in the Survey readliy has access to.

On Friday, I assigned a 7 My span to Unit APP2.2, based on isotopic constraints. This (sub-)zone is split up into three subzones that I though had no isotopic control. So, I arbitrarily assigned ages to them, including an arbitrary age of 284 Ma to the base of APP2.2.2, only to discover that I had taken a sample a few years ago from Denison NS173 that the Boise geochron lab, in Idaho, dated at 282.87 Ma, and that MG Paleo, for two palynology directly above it,  attributed to APP2.2.2.  So, it's important to have this otherwise lost info in there (at least for the time being, until it is published, if ever). This also indicates how reliable the biostrat is, and it is far, far cheaper than isotopic dating. Just about all of the age relationships that you see in strat charts for basins of mid-Paleozoic to Mesozoic age are based on biostrat. So, this will be an important vocab. It will sit up there, at least for QLD, with the timescale. @FAULKNER Susan<mailto:Sue.Faulkner@dnrme.qld.gov.au> please note the paly and isotopic ages from this coal hole.

________________________________
From: David Crosswell <notifications@github.com>
Sent: Monday, 16 November 2020 2:58 PM
To: geological-survey-of-queensland/qldgeofeatures-dataset <qldgeofeatures-dataset@noreply.github.com>
Cc: MCKELLAR John <John.McKellar@dnrme.qld.gov.au>; Mention <mention@noreply.github.com>
Subject: Re: [geological-survey-of-queensland/qldgeofeatures-dataset] Add files via upload (#8)


@johnmckellar<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_johnmckellar&d=DwMCaQ&c=tpTxelpKGw9ZbZ5Dlo0lybSxHDHIiYjksG4icXfalgk&r=7PGzJsKGfMQg6r_oMXGYtVoxRstXoRB0iYsyk0FLe8U&m=5faiBFo4P3XSYKUwdY5o0FfaTY8m-T7ytZsMCJ8dKzg&s=hhYdhDyJ16xJCId1H_N6CqjRLgYRdR0uIMeod5FTtbQ&e=> - is it worth smoothing some of the geojson? Some of it is quite verbose.
Ok if you want to leave it at that granularity.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_geological-2Dsurvey-2Dof-2Dqueensland_qldgeofeatures-2Ddataset_pull_8-23issuecomment-2D727734349&d=DwMCaQ&c=tpTxelpKGw9ZbZ5Dlo0lybSxHDHIiYjksG4icXfalgk&r=7PGzJsKGfMQg6r_oMXGYtVoxRstXoRB0iYsyk0FLe8U&m=5faiBFo4P3XSYKUwdY5o0FfaTY8m-T7ytZsMCJ8dKzg&s=vpJW_S_zyctRiwFcBfmGLq8WLUoMsga7OSAbF-ycI4U&e=>, or unsubscribe<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_AMXRW2W4MWOMMKNMJ6CY7GTSQCWPBANCNFSM4R7ZOLGQ&d=DwMCaQ&c=tpTxelpKGw9ZbZ5Dlo0lybSxHDHIiYjksG4icXfalgk&r=7PGzJsKGfMQg6r_oMXGYtVoxRstXoRB0iYsyk0FLe8U&m=5faiBFo4P3XSYKUwdY5o0FfaTY8m-T7ytZsMCJ8dKzg&s=v3Vofdfh01huKCWhxzkafNEypb9p9XBMeP7rcdUU4xY&e=>.

------------------------------
The information in this email together with any attachments is intended only for the person or entity to which it is addressed and may contain confidential and/or privileged material. There is no waiver of any confidentiality/privilege by your inadvertent receipt of this material. 
Any form of review, disclosure, modification, distribution and/or publication of this email message is prohibited, unless as a necessary part of Departmental business.
If you have received this message in error, you are asked to inform the sender as quickly as possible and delete this message and any copies of this message from your computer and/or your computer system network.
------------------------------

KellyVance commented:
@johnmckellar @DavidCrosswellGSQ 
There may be a disconnect here between a definition and explanatory notes. This level of detail is tending toward an in-depth quasi-article. The definition here should be a short-sharp paragraph that gives the user enough information to distinguish the concept/feature from others within the vocabulary. It shouldn't replace a published work that examines each subject and outlines all the features of the subject and how those were determined.

In my humble opinion I think the comprehensive work John is doing here deserves a proper document or report in its own right that we publish via the Open Data Portal, with a reference in the vocabulary Concept Scheme. Keeping this granularity in a vocabulary means that a) it isn't really fit for purpose for the quick distinctions people look for in a list of concepts like this, and b) it lends itself to being skimmed over, rather than actually read and understood in a way that work this thorough really should.

johnmckellar commented:
Thanks for the feedback David & Vance; in reply:
1.	(David) you could have created the basin groups as collections, but the way they are will work: 

Good!! This has two functions (at least in my head): i. To group them as I would like to see them grouped in GeoResGlobe; and ii, To bring together different basin types, such as basins, depressions, grabens, half-grabens, etc., which will require an addition to the features type vocab. I have not thought about the latter process yet, as to what I need to do.
2.	(David) there are some bad characters in that file. I recommend the VSCode extension Highlight Bad Chars by Kevin Wenger.

3.	Thanks for this. I had been wondering how to get around this issue with hyphens. Vance and/or Kate had identified this issue in the last round, and assumed they were hyphens. I will download the extension to VS, to fix (in the next round).

4.	Verbosity (David , Vance): I will keep definitions to a minimum, and add the rest to notes. My approach to the biostrat vocab has evolved to this point, but I need to go back to the concepts that I started with, to modify accordingly. I have put some detail into notes, so that I do not have to find it again (takes a lot of time), as the vocab can be further built (like the relationships in the geological-timescale ttl, built by Simon Cox & Co.).

Vance, not a bad idea re publishing via the open data portal, but re (for qldgeofeatures): There may be a disconnect here between a definition and explanatory notes: 

In some cases, what’s in the file has to stand in its own right, as, without explanation (which can be confined to scopeNotes), there will be an apparent conflict with what has been published, especially in the case of the Geology of Queensland, which did not get everything right, but it stands as the Bible (which users will use for comparison). And, this is not to mention new data and interpretations.

johnmckellar committed:
Add files via upload

I have fixed-up the bad-character problem. Initially, I thought that it only involved hyphens (which it mainly did), but it also encompassed quotation marks and, in one instance, a series of three full stops ( … ), to indicate that words were left out in a direct quotation. The strange thing about all of this is that this bad character did not occur in all instances of hyphens, etc., only a few.

I down-loaded the ‘highlight bad chars extension’, but it took time to figure out that it was only for VS Code, not for VS (not showing up for the latter as an available extension). Also, the bad character did not show up in VS, with which I prefer to work (as VS puts all of the spatial in a single line, making work with the file much easier for what I was doing with it). After correcting in VS Code, I went back to VS, to make suggested edits (re scopeNotes), and then rechecked in VS Code, for the bad character. Hopefully, there are now all gone.

I have looked at my ‘verbosity’ in scopeNotes. I have reduced some Notes in length, but have otherwise left them as is. I feel that there is insufficient material for a stand-alone, linked document. David C also said that he was happy not to have the info in different places. But, in agreement, I do see the need to follow advice for a linked document for the biostrat vocabs (plural). There is much info and complexity here, and the sands continually shift (as new info comes in).

KellyVance committed:
Update dataset.ttl

Syntax fixes throughout file. Committing directly for expediency.
@johnmckellar I will discuss the fixes I have applied with you separately.
But in the future you can input the file into https://rdf-translator.appspot.com/ which provides a good first pass check and identifies the lines or concepts that have syntax errors.

johnmckellar commented:
Thanks Kate. I will likely not do any further work on it until it is up and running.
